src/cow-react/modules/limitOrders/utils/getOrderSurplus.ts

```diff
@@ -11,62 +10,146 @@ type Surplus = {
 }
 
 /**
- * Calculates SELL surplus based on buy amounts
+ * Calculates SELL surplus
  *
- * @param buyAmount buyAmount
- * @param executedBuyAmount executedBuyAmount
  * @returns Sell surplus
  */
-export function getSellSurplus(buyAmount: BigNumberish, executedBuyAmount: BigNumberish): Surplus {
+export function getSellSurplus(order: Order): Surplus {
```

Comment:
@shoom3301 this one would be a great candidate to move to SDK, right?

Reply:
No the "sellSurplus", but just a getSurplus given an order

Reply:
100%